test/basic.js

```diff
@@ -404,3 +405,22 @@ tape('append and createWriteStreams preserve seq', function (t) {
     })
   })
 })
+
+tape('writes are batched', function (t) {
+  var trackingRam = createTrackingRam()
+  var feed = hypercore(trackingRam)
+  var ws = feed.createWriteStream()
+
+  ws.write('ab')
+  ws.write('cd')
+  ws.write('ef')
+  ws.end(function () {
+    t.deepEquals(trackingRam.log.data, [
+      { write: [ 0, Buffer.from('ab') ] },
+      { write: [ 2, Buffer.from('bcef') ] }
```

Comment:
I think this should be cdef

Reply:
Totally, ... fixed now _(memo: sleep more)_